README.md

```diff
@@ -255,36 +255,37 @@ In diesen Ordner wird die __Datenbank__ des Containers _persistiert_, also dauer
-Das bedeutet: Ebenso wie die Datenbank liegt dein REDAXO dauerhaft auf deinem System und kann von dir bearbeitet werden, während Docker dir nur die notwendige Serverumgebung bereitstellt.
+This directory forms the __Webroot__, which was already mentioned above. It is linked to the directory of the container (a Debian GNU / Linux by the way) in which the Apache web server deposits the website. So if you make adjustments to the REDAXO, they are immediately available to the server, and vice versa.
+This means: Like the database, your REDAXO is permanently on your system and can be edited by you while Docker only provides you with the necessary server environment.
 
-:point_right: _Beachte: Wenn der Ordner beim Start des Containers leer ist, installiert Docker ein frisches REDAXO für dich, und je nach Konfiguration (in `docker-compose.yml`) sogar noch eine Website-Demo dazu. Enthält der Ordner aber bereits Inhalte, ändert Docker nichts daran und startet lediglich den Container._
+
+:point_right: _Keep in mind: If the folder is empty at the start of the container, Docker will install a fresh REDAXO for you, and depending on the configuration (in `docker-compose.yml`) even a website demo. But if the folder already contains content, Docker does not change it and just starts the container._
 
 #### Ignore
 
     .dockerignore
 
-In [dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) wird definiert, welche Dateien und Ordner _nicht_ an den Docker-Daemon überreicht werden. Wenn dein Projektordner sehr voll ist, kannst du die für Docker unwichtigen Daten übergehen und sparst damit Ressourcen.
+In [dockerignore](https://docs.docker.com/engine/reference/builder/#dockerignore-file) defines which files and folders _not_ will be given to the Docker daemon. If your project folder is very busy, you can bypass Docker's unimportant data and save resources.
 
 #### Compose
 
     docker-compose.yml
 
-[Docker Compose](https://docs.docker.com/compose/overview/) ermöglicht dir, __mehrere Container gleichzeitig__ zu starten und zu verknüpfen. Es enthält z. B. Angaben darüber, wie die Container heißen, welche Ports sie benutzen und welche Verzeichnisse deines Systems sie einbinden (Volumes). Zudem kann es Informationen wie Username und Passwörter, in unserem Fall für die Datenbank, enthalten.
+[Docker Compose](https://docs.docker.com/compose/overview/) allows you to start and link __several containers at the same time.__ It contains z. For example, information about what the containers are called, which ports they use, and which directories of your system they are using (volumes). It can also contain information such as username and passwords, in our case for the database.
```

Comment:
[Docker Compose](https://docs.docker.com/compose/overview/) allows you to start and link __several containers at the same time.__ It contains information about what the containers are called, which ports they use, and which directories of your system they are using (volumes). It can also contain information such as username and passwords - in our case for the database.